### PR TITLE
Add support for decimal in `to_json`

### PR DESCRIPTION
**Pull request by andygrove**
Closes https://github.com/NVIDIA/spark-rapids/issues/9872

### CODE REVIEW

**Comment by andygrove** [comment]
build

File: sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala [hunk 1]
```diff
@@ -3600,8 +3600,10 @@ object GpuOverrides extends Logging {
         TypeSig.STRING,
         Seq(ParamCheck("struct",
           (TypeSig.BOOLEAN + TypeSig.STRING + TypeSig.integral + TypeSig.FLOAT +
+            TypeSig.DECIMAL_64 + TypeSig.DECIMAL_128 +
```
Comment (revans2) [inline sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala:3603]: DECIMAL_128 implied DECIMAL_64 so it is redundant to have both here.

**Review by revans2 — APPROVED** [review]
Just a small nit

File: sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala (hunk 1 above)
Comment (andygrove) [reply sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala]: I have removed DECIMAL_64

**Review by revans2 — DISMISSED** [review]

**Comment by andygrove** [comment]
build

**Review by revans2 — APPROVED** [review]

**Pull request merged**